versions/3.0.md

```diff
@@ -221,7 +222,7 @@ Field Name | Type | Description
 <a name="infoTermsOfService"></a>termsOfService | `string` | A URL to the Terms of Service for the API.
 <a name="infoContact"></a>contact | [Contact Object](#contactObject) | The contact information for the exposed API.
 <a name="infoLicense"></a>license | [License Object](#licenseObject) | The license information for the exposed API.
-<a name="infoVersion"></a>version | `string` | **Required** Provides the version of the application API (not to be confused with the specification version).
+<a name="infoVersion"></a>version | `string` | **Required** Provides the version of the application API (not to be confused with the specification version).  While not required, is it suggested that the `info.version` value correspond to the version of the API definition.  In practice, the version of the API _implementation_ may evolve at an entirely different rate.
```

Comment:
`version` is marked **Requried**, then the second sentence starts "While not required". Some may be confused and think this starts a contradiction. I suggest not adding this sentence at all, as it only raises additional questions (is "application API === "API definition"?) Instead, I suggest rephrasing, introducing fewer new/undefined/ambiguous terms:

**Required** Provides the version of the  API (not to be confused with the [_OpenAPI specification version_](#openapi-version-string) or the _implementation_ version which may evolve at different rates).